content/docs/user-guide/managing-external-data.md

```diff
@@ -13,18 +13,16 @@ track data outside of the <abbr>project</abbr>.
 ## How external outputs work
 
 DVC can track existing files or directories on an external location with
-`dvc add` (`out` field). It can also create external files or directories as
-outputs for `dvc.yaml` files (only `outs` field, not metrics or plots).
+`dvc add`. It can also define external outputs for `dvc.yaml` stages to create.
```

Comment:
DVC pipelines stages?

Reply:
Not seeing need to mention pipelines in these docs, we're already going over deps/outs and stages. Too many concepts otherwise, probably? You can click on "stage" to get to pipelines.